src/modules/ekf2/EKF2.cpp

```diff
@@ -920,6 +920,9 @@ void EKF2::PublishAttitude(const hrt_abstime &timestamp)
 		_ekf.getQuaternion().copyTo(att.q);
 
 		_ekf.get_quat_reset(&att.delta_q_reset[0], &att.quat_reset_counter);
+
+		_ekf.getRotVarNed().copyTo(att.variance);
```

Comment:
This is the variance of the rotation around the N-E-D axes. Usually, the quaternion variance is defined in body frame (as in the vehicle_odometry message). I think it makes more sense to separate is as:
- q_variance (in body frame)
- yaw/heading variance (around the Down axis of NED)
- tilt variance (sum of the variances around North and East axes)